src/Microsoft.Diagnostics.Monitoring.EventPipe/Logs/IStateWithTimestamp.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Diagnostics.Monitoring.EventPipe
+{
+    internal interface IStateWithTimestamp
```

Comment:
Is the intent that https://github.com/dotnet/dotnet-monitor/blob/0885d0a80b41e59d36c2a61e31e42c1335215a1e/src/Microsoft.Diagnostics.Monitoring.RestServer/StreamingLogger.cs#L18 will cast the `state` parameter to this interface to get the timestamp and write it out?

Reply:
yep. I will have a PR with that soon.

Reply:
https://github.com/dotnet/dotnet-monitor/pull/256